lib/src/install/osconfig.rs

```diff
@@ -40,6 +56,35 @@ fn test_inject_root_ssh() -> Result<()> {
 
     // The code expects this to exist, reasonably so
     root.create_dir("etc")?;
+    // Test with a symlink
```

Comment:
Super nitpick: maybe worth renaming this helper now to `test_inject_root_ssh_root_symlinked` or something assuming this is what it is testing (i.e. that my other comment that parts of the test are redundant with `test_inject_root_ssh_dir` is true).